dimod/variables.py

```diff
@@ -116,6 +120,25 @@ def count(self, v):
         # everything is unique
         return int(v in self)
 
+    def to_serializable(self):
+        def iter_variables(variables):
```

Comment:
Maybe turn this into a utility function (de-dup from BQM too)?

Reply:
I consider the one in `Variables` the 'definitive' version and the BQM to be the duplication. In the future, I think the BQM will also use the Variables object (e.g. #477) so for now I have left the duplication.

However you are correct that in the mean time we could make this a utility function, probably in `bqm/serialization/utils.py`.

Reply:
I did make it a utility, but I kept in the `Variables` namespace so that we can remove it later without seriously affecting backwards compatibility.